scripts/minikube.sh

```diff
@@ -154,7 +172,20 @@ EXTRA_CONFIG="${EXTRA_CONFIG} --extra-config=kubelet.resolv-conf=${RESOLV_CONF}"
 #extra Rook configuration
 ROOK_BLOCK_POOL_NAME=${ROOK_BLOCK_POOL_NAME:-"newrbdpool"}
 
+kube_latest_version=$(curl -L https://storage.googleapis.com/kubernetes-release/release/stable.txt 2> /dev/null)
```

Comment:
you only need to check `stable.txt` in case `KUBE_VERSION == latest`, otherwise just install `kubectl` with version `KUBE_VERSION`.

Reply:
sure